cli/glnexus_cli.cc

```diff
@@ -53,6 +53,24 @@ static GLnexus::Status LoadYAMLFile(const string& filename, YAML::Node &node) {
     return GLnexus::Status::OK();
 }
 
+static GLnexus::Status check_sanity_multiple_dsals(
+    const string &file1,
+    const string &file2,
+    const vector<pair<string,size_t>> &contigs1,
+    const vector<pair<string,size_t>> &contigs2,
+    const vector<GLnexus::range> &ranges1,
+    const vector<GLnexus::range> &ranges2) {
+
+    if (contigs2.size() != contigs1.size())
+        return GLnexus::Status::Invalid("The number of contigs is different bewteen", file1 + " " + file2);
+
+    if (ranges2.size() != ranges2.size())
```

Comment:
`x != x`

It seems straightforward to just go ahead and verify full equality of the contigs and ranges